### DISCUSSION TITLE
[Feature Request] Add `local` subdomain as a license-key-not-required environment

### DISCUSSION THREAD

**Discussion by bkuhl** [laravel/nova-issues#5447] [Ideas & Feature Requests]
https://nova.laravel.com/docs/4.0/installation.html#using-nova-on-development-staging-domains covers quite a few subdomains already, however we've already established a pattern of using `local` for local environments as all the names currently listed can be used for various hosted environments within a larger organization.  Can the `local` subdomain can be added to the list?  So far this hasn't actually caused any issues preventing us from developing locally, but it would be ideal to avoid such an issue.

**Comment by crynobone** [comment]
> So far this hasn't actually caused any issues preventing us from developing locally, but it would be ideal to avoid such an issue.

`Unregistered` status is just a gentle reminder and doesn't remove/limit any feature.
